lib/mongodb.js

```diff
@@ -436,6 +436,32 @@ MongoDB.prototype.execute = function(model, command) {
       return;
     }
 
+    // MongoDB has deprecated some commands. To preserve
+    // compatibility of model connector hooks, this maps the new
+    // commands to previous names for the observors of this command.
+    switch (context.req.command) {
```

Comment:
Can we use a look-up table instead of a switch statement?

```js
// somewhere above this method

const COMMAND_MAPPING = {
  insertOne: 'insert',
  // ...
};

// here:
const mapped = COMMAND_MAPPING[context.req.command];
if (mapped) context.req.command = mapped;
```